### PR TITLE
Added extra slide for git pull using merge vs rebase

### PR DESCRIPTION
**Pull request by VukanJ**
Added an extra slide which gives a litte bit more insight into the difference between git pull and git pull with rebase. Comments are very welcome

### CODE REVIEW

File: git/content/git.tex [hunk 1]
```diff
@@ -364,6 +359,28 @@
     \end{lstlisting}
 \end{frame}
 
+\begin{frame}{Extra Slide für sauberere Projekt Historien: \texttt{git pull}\quad vs\quad\texttt{git pull --rebase} (optional)}
+    Vielfaches Merging und Merge Konflikte erzeugen eine etwas nichtlineare Projekt-Historie, denn:\\
+    \texttt{git pull} entspricht \texttt{git fetch origin; git merge …} (\to\;gemergter Branch bleibt erhalten)
+
+    Alternativ kann man \texttt{\textbf{git pull --rebase}} ausführen, welches (in etwa) äquivalent ist zu\\\texttt{git fetch origin; git rebase …} (\to\;lokale Commits werden auf neue Commits angewendet).
```
[the remark this answers is not in the document]
Comment (LuckyJosh) [reply git/content/git.tex]: [Gibt Unterschiede in den Details](https://gitolite.com/git-pull--rebase)

File: git/content/git.tex [hunk 2]
```diff
@@ -322,7 +317,7 @@
     \item Merge abschließen:
       \begin{enumerate}
         \item \texttt{git add …}
-        \item \texttt{git rebase --continue}
+        \item \texttt{git commit -m "Merge Konflikt gelöst"}
```
[the remark this answers is not in the document]
Comment (VukanJ) [reply git/content/git.tex]: done

File: git/content/git.tex (hunk 1 above)
[the remark this answers is not in the document]
Comment (VukanJ) [reply git/content/git.tex]: Ok, so should we leave it like this then?

[the remark this answers is not in the document]
Comment (bixel) [reply git/content/git.tex]: I think that should be ok. You could also link @LuckyJosh's link into the `(in etwa)`, but IMHO it is fine as it is.

**Pull request merged**